notices.json

```diff
@@ -200,6 +200,44 @@
       }
     }
   },
+  {
+    "id": "bleve_deprecation",
+    "conditions": {
+      "audience": "sysadmin",
+      "serverVersion": ["<11.0"],
+      "displayDate": ">= 2025-10-16T00:00:00Z",
+      "serverConfig": {
+        "EnableSearching": "true",
```

Comment:
@amyblais we will need QA to test this to ensure this only shows up if BOTH conditions are true (ie Bleve enabled and database search disabled)

If only one condition is true we don't want to display this

Reply:
According to https://github.com/mattermost/mattermost/blob/18eb1347db5dbc21ea9ab760c12efdd8a440ac56/server/channels/app/product_notices.go#L153-L158, all conditions need to be matched.

Reply:
@agarciamontoro What does it mean? Do I need to make changes?